src/help-urls/data/slice.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+import { RequestStatus } from '../../data/constants';
+
+const initialState = {
+  loadingHelpUrlsStatus: RequestStatus.PENDING,
```

Comment:
let's use `RequestStatus.IN_PROGRESS` for loading initial value

Reply:
Why? I don't understand why everyone use `IN_PROGRESS` when logically it's `PENDING`, hadn't started

Reply:
How I can handle if request was started if request in progress in initial state and in progress when really in progess?

Reply:
```
Pending: Work on the objective has not yet started.
In Progress: Work has started toward objective completion
```

Reply:
Actually I agree with you it looks a little confusing)
`frontend-app-course-authoring/src/custom-pages/data/thunks.js`

they use status:
- `RequestStatus.IN_PROGRESS` for loading using `GET`
- `RequestStatus.PENDING` for saving using `POST`, `DELETE`, ...

in this case `slice.js` looks like this
```
loadingStatus: RequestStatus.IN_PROGRESS,
savingStatus: '',
```

Reply:
Ok, I'll change it to `IN_PROGRESS`

Reply:
Alternatively, we can use `''` for both initial
`IN_PROGRESS` for starting loading query 
`PENDING ` for starting saving query